migrations/versions/38f11aa39bff_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: 38f11aa39bff
+Revises: 781e870940ac
+Create Date: 2017-06-01 10:51:38.674997
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '38f11aa39bff'
+down_revision = '781e870940ac'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('sponsors', sa.Column('logo_url', sa.String(), nullable=True))
```

Comment:
Don't add and drop. This will result in loss of data. Do an alter column please.